modules/gapi/src/api/gorigin.cpp

```diff
@@ -21,8 +21,14 @@ cv::GOrigin::GOrigin(GShape s,
 }
 
 cv::GOrigin::GOrigin(GShape s, cv::gimpl::ConstVal v)
-    : shape(s), node(cv::GNode::Const()), value(v), port(INVALID_PORT), kind(cv::detail::OpaqueKind::CV_UNKNOWN)
+    : shape(s), node(cv::GNode::Const()), value(v), port(INVALID_PORT)
```

Comment:
@smirnov-alexey about kind for vectorRef

Reply:
Looks good, thanks you